src/main/webapp/app/exercises/modeling/manage/example-modeling/example-modeling-submission.component.html

```diff
@@ -127,9 +127,9 @@ <h5 *ngIf="toComplete" class="fw-light">{{ 'artemisApp.exampleSubmission.home.as
                     </jhi-modeling-assessment>
                 </div>
 
-                <!--The explanation field is only visible a) in assessment mode, b) for example submissions not used for tutorial and c) when in assessment dashboard (readonly = true)
+                <!--The explanation field is only visible a) in assessment mode, b) for example submissions in read and confirm mode and c) when in assessment dashboard (readonly = true)
                 only if an explanation text is there-->
-                <h3 class="mt-3" *ngIf="assessmentMode && !exampleSubmission?.usedForTutorial && (!readOnly || (assessmentExplanation && assessmentExplanation.length > 0))">
+                <h3 class="mt-3" *ngIf="assessmentMode && selectedMode === 'readConfirm' && (!readOnly || (assessmentExplanation && assessmentExplanation.length > 0))">
```

Comment:
Can‘t we use an Enum here instead of a string?

Reply:
Done in dc0b348e4a1faa469cfda26608f038df5d22275f